plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/AbstractTestDeltaLakeCreateTableStatistics.java

```diff
@@ -1,492 +0,0 @@
-/*
```

Comment:
Could you try preserving the commit history of `AbstractTestDeltaLakeCreateTableStatistics` with preparatory commits?